### ISSUE TITLE
Feature request: See news for a specific country

### ISSUE THREAD

**Issue by KeyboardSounds** [legiblenews/community#14]
I would love to see a country-specific news page so that I can keep track of events that have the biggest impact on my life— eg. https://legiblenews.com/australia to see Australian news.

Ideally, this would have the same structure as the standard daily Legible News page, with the same topic headings, but "filtered" by country.

**Comment by bradgessler** [comment]
Great idea! I'd generally like to get more "editions" of legible news, but I'd need to figure out a way to source that content in the most objective way possible.

To be honest, I don't expect other editions of the news to be available anytime soon, but it is something I'm thinking about both at a country level and a more local level, like metro area.

As far as filtering goes, I think what you mean is you want to see an issue of the news for all of Australia. I'd call that an Australian issue of the news, and not filtered.

The reason I think that distinction is important is because filtering news suggests hiding other stuff, which is something that I think is a problem with the news these days. Legible News will never filter, but it would produce daily and weekly issues of the news for a specific geography.

**Comment by KeyboardSounds** [comment]
> As far as filtering goes, I think what you mean is you want to see an issue of the news for all of Australia. I'd call that an Australian issue of the news, and not filtered.
> The reason I think that distinction is important is because filtering news suggests hiding other stuff, which is something that I think is a problem with the news these days. Legible News will never filter, but it would produce daily and weekly issues of the news for a specific geography.

Yes! That's a much better way of thinking about it it. Something like the AP Media API: https://developer.ap.org/ap-media-api/ or the  Guardian Open Platform https://open-platform.theguardian.com/access/ might be useful— picking APIs from sources that score well on https://adfontesmedia.com/

**Comment by reinaldotmorais** [comment]
I need one for Brazil (urgently!), especially in the weekly edition.

**Comment by ghost** [comment]
Yes! A US one would be great.
